gatsby-config.js

```diff
@@ -35,6 +35,31 @@ module.exports = {
         path: path.join(__dirname, 'content/uxcs')
       }
     },
+    {
+      resolve: `gatsby-source-filesystem`,
+      options: {
+        name: `components`,
+        path: path.join(__dirname, 'content/components')
+      }
+    },
+    {
+      resolve: `gatsby-source-filesystem`,
+      options: {
+        name: `react-components`,
+        path: path.join(__dirname, 'react-components/packages'),
+        ignore: ['**/*.!(ts|tsx)']
+      }
+    },
+    'gatsby-source-news',
+    'gatsby-source-react-packages',
```

Comment:
It turns out ordering matters for these `source` plugins. This ensures that all `react-packages` and `news` are sourced before page-level GraphQL queries.